### ISSUE TITLE
Issue 3: Missing focus ring outline

### ISSUE THREAD

**Issue by keerthanapareddy** [wenqili/adjacent#36]
**Describe the bug**
While tabbing through articles on the home page there is focus ring outline on article headings, then it goes missing, then it comes back on the next articles heading. 

**To Reproduce**
1. Go to 'https://itp.nyu.edu/adjacent/issue-3/'
2. Press the tab key until you reach the articles
3. See error

**Expected behavior**
The missing focus ring should be fixed.

**Screenshots**
<img width="1440" alt="Screen Shot 2019-04-19 at 9 59 35 AM" src="https://user-images.githubusercontent.com/31640804/56428526-b2221680-628d-11e9-9a6d-35fbe13b1489.png">

missing focus ring but is on a link:
<img width="1440" alt="Screen Shot 2019-04-19 at 9 59 43 AM" src="https://user-images.githubusercontent.com/31640804/56428548-c0703280-628d-11e9-8c8c-504992913a65.png">

Then goes to the next article:
<img width="1440" alt="Screen Shot 2019-04-19 at 9 59 51 AM" src="https://user-images.githubusercontent.com/31640804/56428559-cbc35e00-628d-11e9-9af5-33a379725157.png">

**Issue (please complete the following information):**
 - Issue 03

**Platform (please complete the following information):**
 - All browsers
